resources/compass-runtime-agent/templates/tests/compass-external-solution-test.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: "testing.kyma-project.io/v1alpha1"
+kind: TestDefinition
+metadata:
+  name: {{ .Chart.Name }}-external-solution
+  namespace: {{ .Release.Namespace }}
+  labels:
+    kyma-project.io/test-kind: e2e
+    app: {{ .Chart.Name }}-external-solution-test
+    app.kubernetes.io/name: {{ .Chart.Name }}-external-solution-test
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+spec:
+  disableConcurrency: false
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+    spec:
+      restartPolicy: Never
+      serviceAccountName: {{ .Chart.Name }}-external-solution-test
+      {{ if .Values.global.isLocalEnv }}
+      hostAliases:
+        - ip: {{ .Values.global.minikubeIP }}
+          hostnames:
+          - "counter-service.{{ .Values.global.ingress.domainName }}"
+          - "gateway.{{ .Values.global.ingress.domainName }}"
+          - "compass-gateway.{{ .Values.global.ingress.domainName }}"
+          - "dex.{{ .Values.global.ingress.domainName }}"
+      {{ end }}
+      containers:
+        - name: tests
+          image: {{ .Values.global.images.containerRegistry.path }}/{{ .Values.global.images.compassExternalSolutionTests.dir }}e2e-external-integration-test:{{ .Values.global.images.compassExternalSolutionTests.version }}
+          imagePullPolicy: Always
+          command:
+            - "bin/sh"
+          args:
+            - "-c"
+            - |
+              echo 'TESTING start'
+              sleep 10
```

Comment:
Suggestion: It would be better to check if Director is ready. Waiting 10 seconds can be not predictable.

However, this is an idea as a follow-up task.